soccer/src/soccer/planning/planner_node.cpp

```diff
@@ -142,6 +142,13 @@ PlannerForRobot::PlannerForRobot(int robot_id, rclcpp::Node* node,
         [this](rj_msgs::msg::RobotStatus::SharedPtr status) {  // NOLINT
             had_break_beam_ = status->has_ball_sense;
         });
+
+    hypothetical_path_service_ = node_->create_service<rj_msgs::srv::PlanHypotheticalPath>(
+        fmt::format("hypothetical_trajectory_robot_{}", robot_id),
+        [this](const std::shared_ptr<rj_msgs::srv::PlanHypotheticalPath::Request> request,
```

Comment:
is there a reason this can't be a const reference?
```suggestion
        [this](const std::shared_ptr<rj_msgs::srv::PlanHypotheticalPath::Request> request&,
```

Reply:
like weird ROS spec or smth?

Reply:
I think so bad things happen when I add the "&"